### PR TITLE
Update AZ400_M10_Controlling_Deployments_using_Release_Gates.md

### PR DESCRIPTION
**Pull request by MOC-Labs-Review**
**Replace issue name M00-LAB00:QUICK_DESCRIPTION, for example "M01-LAB01: My new issue" (or same name as linked Issue)**

## Related Issue

**Link related Github Issue** 🢂 Fixes # . (Include issue number after #)

## Checklist 
Mark completed with "x" between brackets, "[x]":
- [ ] Has related GitHub Issue 💥 [Create Issue](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#reporting-issues) 📝
- [ ] Tested it
- [ ] Read the PR collaboration guide 👓 [Collaboration Guide](https://github.com/MicrosoftLearning/AZ400-DesigningandImplementingMicrosoftDevOpsSolutions/blob/master/.github/CONTRIBUTING.md#pull-requests) 📝

Changes proposed in this pull request:

-
-
-

### CODE REVIEW

**Comment by MOC-Labs-Review** [comment]
Submitted by garjen55 on behalf of MOC Labs Review

File: Instructions/Labs/AZ400_M10_Controlling_Deployments_using_Release_Gates.md [hunk 1]
```diff
@@ -155,7 +155,7 @@ In this task, you will create two Azure web apps representing the **Canary** and
 1.  In the Azure portal, navigate to the resource group **az400m10l01-RG** you created in the previous task.
 1.  In the list of resources, click the **Canary** web app. 
 1.  On the **Canary** web app page, in the vertical menu on the left, in the **Settings** section, click **Application Insights**.
-1.  On the **Application Insights** blade, click **Turn on Application Insights**.
+1. On the **Application Insights** blade, enable **Collect application monitoring data using Application Insights**.
```
Comment (unaihuete93) [inline Instructions/Labs/AZ400_M10_Controlling_Deployments_using_Release_Gates.md:158]: I still see "Turn On Application Insights". Can you provide more information (screenshots)?

File: Instructions/Labs/AZ400_M10_Controlling_Deployments_using_Release_Gates.md [hunk 2]
```diff
@@ -199,7 +199,7 @@ In this task, you will update release tasks.
 
     > **Note**: The canary environment has 2 tasks which, respectively, publish the package to Azure Web App and enable continuous monitoring of the application after deployment. 
 
-1.  On the **All pipelines > PartsUnlimited-CD** pane, ensure that the **Canary Environment** stage is selected. In the **Azure subscription** dropdown list, select your Azure subscription and click **Authorize**. If prompted, authenticate by using the user account with the Owner role in the Azure subscription.
+1. On the **All pipelines > PartsUnlimited-CD** pane, ensure that the **Canary Environment** stage is selected. In the **Azure subscription** dropdown list, select your Azure subscription, shown under **Available Azure Service connections**, and click **Authorize**. If prompted, authenticate by using the user account with the Owner role in the Azure subscription.
```
Comment (unaihuete93) [inline Instructions/Labs/AZ400_M10_Controlling_Deployments_using_Release_Gates.md:202]: This is wrong. "Service Connections" are already authorized connections, they wont be prompted to "Authorize". They need to create a new Service Connection for the project, which is done as explained, choosing it from "available Azure Subscriptions" and Authorizing.

**Comment by unaihuete93** [comment]
Closing until feedback is given.

**Pull request closed without merging**